lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb

```diff
@@ -14,19 +14,27 @@ def initialize(icn)
 
   # @param [string] lighthouse_client_id: the lighthouse_client_id requested from Lighthouse
   # @param [string] lighthouse_rsa_key_path: path to the private RSA key used to create the lighthouse_client_id
-  def get_rated_disabilities(lighthouse_client_id, lighthouse_rsa_key_path)
-    auth_params = {
-      launch: Base64.encode64(JSON.generate({ patient: @icn }))
-    }
-    data = @service.get_rated_disabilities(
-      lighthouse_client_id,
-      lighthouse_rsa_key_path,
-      { auth_params: }
-    )
+  # @return [integer] the combined disability rating
+  def get_combined_disability_rating(lighthouse_client_id, lighthouse_rsa_key_path)
+    data = data(lighthouse_client_id, lighthouse_rsa_key_path)
```

Comment:
should we overwrite the `data` method with the `data` variable?
maybe rename the method to `get_data` or `raw_data`? :thinking:

Reply:
yes - just updated the `data` method to `get_data`.